django/utils/autoreload.py

```diff
@@ -220,6 +220,12 @@ def get_child_arguments():
     py_script = Path(sys.argv[0])
 
     args = [sys.executable] + ['-W%s' % o for o in sys.warnoptions]
+    if hasattr(sys, "implementation") and sys.implementation.name == "cpython":
+        xoptions_args = [
+            "-X%s" % key if isinstance(value, bool) else "-X%s=%s" % (key, value)
+            for key, value in sys._xoptions.items()
+        ]
+        args = args + xoptions_args
```

Comment:
We currently use single quotes and this can be condensed a little and use f-strings.

We can also drop the `hasattr()` check as the [`sys.implementation` docs](https://docs.python.org/3/library/sys.html#sys.implementation) state that it _must_ exist.

```suggestion
    if sys.implementation.name == 'cpython':
        args.extend(
            f'-X{key}' if isinstance(value, bool) else f'-X{key}={value}'
            for key, value in sys._xoptions.items()
        )
```

Reply:
Got, I see !